docs/get_started/roadmap.md

```diff
@@ -0,0 +1,231 @@
+:::
+
+To get better answers, ask questions better. Here are some tips for asking questions:
+
+* Only ask for a decisions when you can't make it yourself. Before you pose a question to the decisionmaker, ask your interview-building team (or the community).
+* Avoid open-ended questions. When you decide to pose a question to the decisionmaker, give them two or three options and explain the pros, cons, and implications of each.
+* If you do ask an open-ended question, explain that you are trying to generate ideas, not decide on a solution.
+* Don't ask the decisionmaker to design the interview. It is usually better to ask for goals ("Do you need the attorney's ID number?"), not details ("Do you want the ID number field to show up right under the attorney's name field?").
+
+**Remember: if you ask a question you will get an answer.** If you don't need an answer, asking for one is not likely to help and may complicate the project.
+
+Finally, when showing the relevant part of the interview as part of asking a question, remind the decisionmaker and stakeholders that the interview is a work in progress and you are not ready for feedback beyond the answer to your question.
+
+## Get Feedback on the Interview
+
+Once the interview works from start to finish and you have closed all the issues that are in scope for the [MVP](#stick-to-an-mvp), it is ready for feedback. Start by getting preliminary feedback from someone with Document Assembly Line experience. After you have made revisions based on the preliminary feedback, give the interview to the decisionmaker and stakeholders for their feedback.
+
+:::tip
+For complex interviews that involve multiple forms/templates, consider doing this in stages. Start with the simplest form and get preliminary and stakeholder feedback early. Then move on to the most complex form to confirm the shape and logic of the overall interview. Then continue with the remaining forms.
+:::
+
+### Preliminary Feedback
+
+Before you show the interview to the decisionmaker and stakeholders, get someone with Document Assembly Line experience to test it with you. This will help you identify issues you may have missed and questions you still need to ask the decisionmaker.
+
+To get the interview ready:
+
+1. [Merge your code](../github#create-a-pull-request) into **main** or a testing branch
+2. [Create a new playground project](../github#create-and-manage-playground-projects) for the interview demonstration and pull your code into it
+3. Make sure the interview works as expected in the testing project
+
+Once the interview is ready, schedule a video meeting with the tester so you can watch them go through the interview. Or they can watch you demonstrate it. Record the test if you can so you can focus on the test instead of taking notes.
```

Comment:
This is a template for *other* courts and orgs who want to learn from the LIT Lab's way of doing things. It's a roadmap, not a tutorial.

With that in mind, this is probably too much detail.

Reply:
Maybe at least giving some kind of link instead of just saying, "Do user testing". Or at least acknowledging that that it's a complex thing all of it's own that we're not going into here. Right now it feels like a "3. Draw the rest of the owl" type of item.